app/src/main/java/swati4star/createpdf/activity/RearrangePdfPages.java

```diff
@@ -72,8 +75,10 @@ private void initRecyclerView(ArrayList<Bitmap> images) {
         mRearrangeImagesAdapter = new RearrangePdfAdapter(this, images, this);
         recyclerView.setAdapter(mRearrangeImagesAdapter);
         mSequence = new ArrayList<>();
-        for ( int i = 0; i < images.size(); i++)
+        for ( int i = 0; i < images.size(); i++) {
             mSequence.add(i + 1);
+        }
+        mInitialSequence.addAll(mSequence);
```

Comment:
There is `ArrayList<Bitmap> bitmaps = new ArrayList<>();` in `RemovePagesFragment` file.
Can't this be used for comparison rather than this approach?
See line 203 in `RemovePagesFragment`

Reply:
@vyankatesh24 What is the advantage of this approach above the implemented one?

Reply:
As ArrayList is already present there. you can directly compare rather this overhead of creating a new one and then comparing.

Reply:
But the bitsmap is present in different activity so we will have to send it as an intent to the RearrangePdfPages activity and it can cause a problem if pdf size is large.

Reply:
Ok, then let your approach be as it is.
Make other suggested changes :+1:

Reply:
Ok 
Thanks